### ISSUE TITLE
Release of the interference code for a single ply

### ISSUE THREAD

**Issue by Pamogar** [JonasSchult/Mask3D#46]
Congrats on your great work and thank you for releasing it in open source.
Is there any news about the release of the interference code used in the demo? It would be very convenient to use it with our own pointclouds.

Thank you in advance!

**Comment by CharlesCai123** [comment]
Yes, the same to me. The author has made remarkable achievements on 3D Semantic Instance Segmentation. And we wish to obtain an inference file for replication and application on our dataset. Thanks!

**Comment by IvanGarcia7** [comment]
@CharlesCai123 Where can i find more information about how to create a custom dataset to use Mask3d?

**Comment by ghy0324** [comment]
@Pamogar @Charlescai123 @IvanGarcia7 Hi! I also have this requirement, have you successfully run inference on your point cloud?

**Comment by tangze5258** [comment]
Has the problem been solved, bros?

**Comment by ghy0324** [comment]
> Has the problem been solved, bros?

@tangze5258 You may refer to [this version](https://github.com/cvg/Mask3D).
